uaclient/entitlements/repo.py

```diff
@@ -82,7 +79,7 @@ def enable(self, *, silent_if_inapplicable: bool = False) -> bool:
                     ['apt-get', 'install', '--assume-yes'] + self.packages,
                     capture=True, retry_sleeps=APT_RETRIES)
             except util.ProcessExecutionError:
-                self.disable(silent=True, force=True)
+                self._cleanup()
```

Comment:
Isn't this _cleanup  still called from fips on failure to install one of fips packages?